services/constants.js

```diff
@@ -5,5 +5,11 @@ module.exports = Object.freeze({
 	},
 	eventTypes: {
 		update_targeting: 'UPDATE_TARGETING'
+	},
+	bigQueryTables: {
```

Comment:
since bigQuery exporting is a side functionality that may be moved out of this repo, can we get rid of this?